apps/fluent-tester/src/E2E/Shadow/pages/ShadowPageObject.ts

```diff
@@ -0,0 +1,17 @@
+import { SHADOW_TESTPAGE, HOMEPAGE_SHADOW_BUTTON } from '../../../TestComponents/Shadow/consts';
+import { BasePage, By } from '../../common/BasePage';
+
+class ShadowTestPage extends BasePage {
+  /*****************************************/
+  /**************** Getters ****************/
+  /*****************************************/
+  get _testPage() {
```

Comment:
I believe there should be another constant here? `_pageName`.

This will cause error's in the error messages

Reply:
Turns out the _pageName wasn't the issue, but rather that I was calling the wrong page object to go to the shadow test page.